flask_app.py

```diff
@@ -299,7 +374,11 @@ def get_token(code):
 def get_username(access_token):
     headers = {"Authorization": "bearer " + access_token, 'User-agent': 'CFB Risk Orders'}
     response = requests.get(REDDIT_ACCOUNT_URI, headers=headers)
-    me_json = response.json()
+    # If reddit or discord fail, then don't return anything
+    try:
+        me_json = response.json()
+    except:
+        return None
     return me_json['name']
```

Comment:
This shouldn't be necessary if `check_identity_or_auth()` is rewritten, but still probably a good check. It could be shorter though.

```suggestion
    # If reddit or discord fail, then don't return anything
    try:
        return response.json()['name']
    except:
        return None
```